app/src/frontend/building/data-containers/category-view-props.ts

```diff
@@ -0,0 +1,21 @@
+interface CopyProps {
+    copying: boolean;
+    toggleCopying: () => void;
+    toggleCopyAttribute: (key: string) => void;
+    copyingKey: (key: string) => boolean;
+}
+
+interface CategoryViewProps {
+    intro: string;
+    building: any; // TODO: add Building type with all fields
```

Comment:
even though there is a new `Building` type, in order for this to be used in the category views, it would need to have all attribute fields in the type. This would then require adding the attributes to the building type whenever a new attribute is added. We need to think if this is the best way to go, or if there would be one that would require less modifications in code to add a new category.
The current state is easier to code but it's at the cost of not having stricter type checking.